iredis/entry.py

```diff
@@ -204,13 +208,14 @@ def gather_args(ctx, h, p, n, password, newbie, iredisrc, decode, raw, rainbow,
     \b
     Examples:
       - iredis
+      - iredis dsn
```

Comment:
This will conflict with `iredis get foo`. Mycli can do this because it read sql from stdin, like this `mycli < select * from foo`.

How about changing it to an option?